pkg/apis/workflow/v1alpha1/http_types.go

```diff
@@ -21,6 +21,8 @@ type HTTP struct {
 	Headers []HTTPHeader `json:"headers,omitempty" protobuf:"bytes,3,rep,name=headers"`
 	// TimeoutSeconds is request timeout for HTTP Request. Default is 30 seconds
 	TimeoutSeconds *int64 `json:"timeoutSeconds,omitempty" protobuf:"bytes,4,opt,name=timeoutSeconds"`
+	// SuccessCondition is an expression if evaluated to true is considered successful
+	SuccessCondition string `json:"successCondition,omitempty"`
```

Comment:
@alexec `make codegen -B` is not adding the protobuf types or editing the protobuf files. Did something change here?

Reply:
Resolved